array_api_compat/torch/_aliases.py

```diff
@@ -694,7 +695,7 @@ def take(x: array, indices: array, /, *, axis: Optional[int] = None, **kwargs) -
            'atan2', 'bitwise_and', 'bitwise_left_shift', 'bitwise_or',
            'bitwise_right_shift', 'bitwise_xor', 'divide', 'equal',
            'floor_divide', 'greater', 'greater_equal', 'less', 'less_equal',
-           'logaddexp', 'multiply', 'not_equal', 'pow', 'remainder',
+           'logaddexp', 'multiply', 'newaxis', 'not_equal', 'pow', 'remainder',
```

Comment:
I've tried to keep these in the same order they are in the file (although I see now that that isn't really completely true).